frontend/content/info-tiles.ts

```diff
@@ -13,7 +13,7 @@ export const INFO_TILES_CONTENT = [
   },
   {
     front: 'Jakie **nagrody** można zdobyć podczas wydarzenia?',
-    back: 'Pula nagród cały czas rośnie, ale już teraz możemy zagwarantować, że na najlepsze drużyny czekają atrakcyjne nagrody — będą one wyjątkowym wyróżnieniem za wysiłek, zaangażowanie i umiejętności zaprezentowane podczas rywalizacji.',
+    back: 'Pula nagród edycji dla szkół podstawowych wynosi **1000 złotych**. Każdy uczestnik może liczyć również na pamiątkowe upominki. Pula nagród ciągle rośnie, a najlepsze drużyny mogą liczyć na wyjątkowe wyróżnienia za swoje zaangażowanie i umiejętności!',
```

Comment:
```suggestion
    back: 'Pula nagród edycji dla szkół podstawowych obecnie wynosi **1000 złotych** i wciąż rośnie! Każdy uczestnik może liczyć również na pamiątkowe upominki. Najlepsze drużyny mogą liczyć na wyjątkowe wyróżnienia za swoje zaangażowanie i umiejętności!',
```

Or something like that, but the important thing is to mention `wciąż rośnie` in the first sentence